web/src/pill_typeahead.ts

```diff
@@ -89,7 +89,7 @@ export function set_up_stream(
     new Typeahead(bootstrap_typeahead_input, {
         items: 5,
         dropup: true,
-        helpOnEmptyStrings: true,
+        helpOnEmptyStrings: opts.help_on_empty_strings,
```

Comment:
Is this change unrelated? Looks like a good change though :)

Reply:
Both commits are related to the PR #28686.

Reply:
Oh oops I didn't see it was in a separate commit, perfect :)